app/src/pages/upload/confirmation.tsx

```diff
@@ -29,21 +56,27 @@ const Confirmation: NextPage = () => {
         <p className="usa-intro measure-5">
           Thank you for uploading your documents. Here are the results:
         </p>
-        <ul>
+        <ul className="padding-left-1">
           {parsedResults?.results?.map((result, idx) => (
-            <li key={idx}>
+            <li key={idx} className="usa-icon-list__item">
               {result.value && (
                 <>
-                  {result.value.imagePath}:{""}
-                  {result.value.isBlurry ? "blurry!" : "sharp"} (
-                  {result.value.score})
+                  {generateBlurryIcon(result.value.isBlurry)}
+                  <span className="usa-icon-list__content padding-top-2px">
+                    {" "}
+                    {formatImagePath(result.value.imagePath)}
+                    {result.value.isBlurry ? " is blurry" : ""}
+                  </span>
                 </>
               )}
               {result.reason && (
```

Comment:
nit: It took me a while to read the code to realize where `result.value` and `result.reason` were coming from. I had an "aha" moment when I realized that they are probably promises and the "reason" is for the rejected promise.

It seems a bit like a code smell to me to use promise rejection to indicate a business-logic decision (document is blurry), as potentially other errors could trigger the rejection and we would want those to have different error handling. At some point, maybe worth a refactor (but not a big deal for now since it's working).